SWLOR.Game.Server/Feature/ShipModuleDefinition/IonCannonModuleDefinition.cs

```diff
@@ -21,18 +21,25 @@ public class IonCannonModuleDefinition : IShipModuleListDefinition
             IonCannon("ion_cann_2", "Ion Cannon II", "Ion Cann. II", "Deals 17 EM DMG to your target.", 3, 5f, 12, 17);
             IonCannon("ion_cann_3", "Ion Cannon III", "Ion Cann. III", "Deals 21 EM DMG to your target.", 4, 6f, 15, 21);
             IonCannon("ion_cann_4", "Ion Cannon IV", "Ion Cann. IV", "Deals 26 EM DMG to your target.", 5, 7f, 18, 26);
+            IonCannon("cap_turbo_1", "Capital Ship Turbolaser I", "Cap Turbo I", "A corvette's main guns. 35 DMG.", 5, 15f, 20, 45);
```

Comment:
There are a lot of duplicates here. If the stats are exactly the same, there's no reason to introduce two of the same type of item. Please remove the duplicates and rename the items accordingly.

Reply:
These are less duplicates and more different recast. For instance, firing once every 2.5 rounds (15f) vs every 2 rounds (12f). It's a 20% DPR change. I felt that difference in recast time was better than just making bigger ships fire bigger guns every time. That said I might change it to 15f and 9f to differentiate them better.